Presentation.Web/Controllers/External/V2/ItSystemV2Controller.cs

```diff
@@ -191,6 +194,7 @@ public IHttpActionResult PostItSystem([FromBody] RightsHolderCreateItSystemReque
 
         /// <summary>
         /// Sets IT-System values
+        /// If a property value is not provided, KITOS will fallback to the default value of the type and that will be written to the it-system so remember to define all data from the contract.
```

Comment:
contract er måske lidt misvisende eftersom der også eksisterer contracts i KITOS. Måske "in the request DTO" i stedet?

Reply:
ok. god pointe